tools/ectyper/ectyper.xml

```diff
@@ -41,19 +41,19 @@
   ]]>
   </command>
   <inputs>
-    <param name="input" type="data"  format="fastq,fasta" label="Genome(s) input(s)" help="FASTA or FASTQ file(s) with contig(s)"/>
-    <param name="mash_input" type="data" optional="true" format="binary" label="Mash genome sketches (Optional)" help="Optionally provide MASH sketches to find closest genome (in case O/H typing fails)"/>
+    <param name="input" type="data"  format="fastq,fasta" label="Genome(s) input(s)" help="FASTA or FASTQ file(s)"/>
+    <param name="mash_input" type="data" optional="true" format="binary" label="Mash genome sketches (Optional)" help="Optionally provide custom MASH genome sketch to help with species identification (otherwise default RefSeq sketch is used)"/>
     <section name="adv_param" title="Advanced parameters" expanded="False">
       <param name="min_percentIdentity" type="integer" value="90" min="1" max="100"/>
-      <param name="percentLength" type="integer" value="50" min="1" max="100"/>
+      <param name="percentLength" type="integer" value="10" min="1" max="100"/>
       <param name="verifyEcoli" type="boolean" checked="true" label="Enable E. coli species verification"/>
       <param name="alleleSequence" type="boolean" checked="false" label="Print the allele sequences as the final columns of the output?"/> 
       <param name="logging" type="boolean" checked="false"  label="Include log file in the run outputs?" />
     </section>  
   </inputs>
   <outputs>
-    <data name="output_result" format="tabular" from_work_dir="output.tsv" label="${tool.name} serotype report"> </data> 
-    <data name="output_log" format="text" from_work_dir="ectyper.log" label="${tool.name} log file"> 
+    <data name="output_result" format="tabular" from_work_dir="output.tsv" label="${tool.name} serotype report on ${input.element_identifier}"> </data>
+    <data name="output_log" format="text" from_work_dir="ectyper.log" label="${tool.name} log file on ${input.element_identifier}">
```

Comment:
Can you change this from `format="text"` to `format="txt"`, which will enable the file to display within Galaxy.

Reply:
Done, thank you!